src/core/mount.c

```diff
@@ -195,6 +195,13 @@ static int mount_arm_timer(Mount *m, usec_t usec) {
 
         assert(m);
 
+        if (usec == USEC_INFINITY) {
+                if (m->timer_event_source)
+                        return sd_event_source_set_enabled(m->timer_event_source, SD_EVENT_OFF);
+
+                return 0;
```

Comment:
One nitpick: `sd_event_source_set_enabled()` is a noop if `m == SD_EVENT_OFF && !s`. So please collapse this to:
```C
if (usec == USEC_INFINITY)
   return sd_event_source_set_enabled(m->timer_event_source, SD_EVENT_OFF);
```

Reply:
I force-pushed with this change. I hope that's OK.

Reply:
Thanks.